components/Carousel/Carousel.js

```diff
@@ -1,14 +1,66 @@
-class Carousel {
+function makeCarousel() {
+    let index2 = 0;
+
+    class Carousel {
+        constructor(element) {
+            this.element = element;
+            this.rightBtn = document.querySelector(".right-button");
+            this.leftBtn = document.querySelector(".left-button");
+            this.images = document.querySelectorAll(".carousel img");
+
+            this.rightBtn.addEventListener("click", () => { this.selectRight() });
+            this.leftBtn.addEventListener("click", () => { this.selectLeft() });
+
+
+        }
+        selectRight() {
+            if (index2 < 3) {
```

Comment:
A ternary statement here would have made this code a lot more compact, but it is not at all necessary.